app/api/resources/rapid_view.py

```diff
@@ -0,0 +1,64 @@
+from urllib.parse import parse_qs
+from fastapi import APIRouter, Request, HTTPException, Query
+from fastapi.responses import RedirectResponse
+import logging
+import re
+from core.logging import InterceptHandler
+from core.config import ENSEMBL_URL
+from api.utils.metadata import get_genome_id_from_assembly_accession_id
+from api.utils.rapid import format_assembly_accession
+
+logging.getLogger().handlers = [InterceptHandler()]
+
+router = APIRouter()
+
+
+# Resolve rapid urls
+@router.get("/{species_url_name}", name="Rapid Species Home")
+@router.get("/{species_url_name}/", name="Rapid Species Home")
+@router.get("/{species_url_name}/{subpath:path}", name="Rapid Species Home")
+async def resolve_species(
```

Comment:
Now that this route is redirecting to a lot of different pages, you may want to change the value of the `name` parameter on lines 17-19 to something more generic.